crates/matrix-sdk-crypto/src/machine/mod.rs

```diff
@@ -1992,6 +2011,17 @@ impl OlmMachine {
         self.inner.identity_manager.update_tracked_users(users).await
     }
 
+    /// Mark all tracked users as dirty.
+    ///
+    /// See `IdentityManager::mark_all_tracked_users_as_dirty()` to learn
```

Comment:
IdentityManager isn't `pub`, besides the docs on `IdentityManager::mark_all_tracked_users_as_dirty()` also points you to another non-pub struct.

I'm afraid you'll have to document this without indirection right here.

Reply:
Done!